parser_test.go

```diff
@@ -183,6 +185,56 @@ func TestParser_ParseTransactions(t *testing.T) {
 	}
 }
 
+func TestParser_GetBaseFee(t *testing.T) {
+	tests := []struct {
+		name    string
+		version string
+		url     string
+		height  string
+	}{
+		{
+			name:    "parser with getBaseFee",
+			version: v2.NodeVersionsSupported[0],
+			url:     nodeUrl,
+			height:  "2907480",
+		},
+		{
+			name:    "parser with getBaseFee fallback",
+			version: v2.NodeVersionsSupported[0],
+			url:     nodeUrl,
+			height:  "3140097",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			lib := getLib(t, tt.url)
+
+			tipset, err := readTipset(tt.height)
+			require.NoError(t, err)
+			ethlogs, err := readEthLogs(tt.height)
+			require.NoError(t, err)
+			traces, err := readGzFile(tracesFilename(tt.height))
+			require.NoError(t, err)
+
+			p, err := NewFilecoinParser(lib, getCacheDataSource(t, tt.url), nil)
+			require.NoError(t, err)
+			txs, adds, err := p.ParseTransactions(traces, tipset, ethlogs, types.BlockMetadata{NodeInfo: types.NodeInfo{NodeMajorMinorVersion: tt.version}})
```

Comment:
The GetBaseFee method is not used on ParseTransactions fn. We should use it directly in order to check we are actually getting the base fee from traces or tipset!